misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_10.bal

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// check this
+
+import ballerina/jballerina.java as _;
```

Comment:
I tried to re-format this already formatted code. It resulted in a different output. We need to have a consistent behavior across multiple formats.

<img width="864" alt="Screenshot 2024-04-02 at 16 17 00" src="https://github.com/ballerina-platform/ballerina-lang/assets/39232462/50c7de65-7249-4264-a1dc-7310b974ff44">

Reply:
btw not mandatory may be we can include it later.  We can have a separate formatter test; re-formatting the existing "assert" bal files in the formatter test directory. It would make sure multiple formats wouldn't break the formatted code.